src/yang/ietf-bgp-rib.yang

```diff
@@ -379,6 +392,27 @@ submodule ietf-bgp-rib {
           uses ext-community-attributes;
         }
       }
+      container large-communities {
+        description
+          "Enclosing container for the list of large community
+           attribute sets.";
+        list large-community {
+          key "index";
+          config false;
+          description
+            "List of path attributes that may be in use by multiple
+             routes in the table.";
+          leaf index {
+            type uint64;
+            description
+              "System generated index for each attribute set.  The
```

Comment:
Two spaces after the period.

Reply:
That is already two spaces?

Reply:
Sorry, I meant to say there are two spaces. Did you want two spaces, or reduce it to one?